CMakeLists.txt

```diff
@@ -20,12 +20,19 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/rpcs3/cmake_modules")
 set(CMAKE_CXX_STANDARD 20)
 include(CheckCXXCompilerFlag)
 
-if(NOT CMAKE_BUILD_TYPE)
-    message(STATUS "No build type selected, default to Release")
+# get build type
+if(CMAKE_BUILD_TYPE MATCHES "Release")
+    message(STATUS "Release build seleceted.")
+    set(CMAKE_BUILD_TYPE "Release")  
```

Comment:
This doesn't make any sense. The var is already "Release".

Reply:
if you try and set an invalid build type as a build option, itll break the cmake build. this change means that if the build type is invalid it will default to release

Reply:
Need RelWithDebInfo type, and MinSizeRel for completeness. Otherwise they will default to Release.